docker/apache-ignite/run.sh

```diff
@@ -20,7 +20,7 @@ if [ ! -z "$OPTION_LIBS" ]; then
   IFS=, LIBS_LIST=("$OPTION_LIBS")
 
   for lib in ${LIBS_LIST[@]}; do
-    cp -r $IGNITE_HOME/libs/optional/"$lib"/* \
+    cp -r $IGNITE_HOME/libs/optional/"$lib" \
```

Comment:
Won't be better to write it down as `cp -r "${IGNITE_HOME}/libs/optional/$lib"`? `${IGNITE_HOME}` potentially can have spaces or other special symbols as well.

Reply:
Since `$IGNITE_HOME` specified in `Dockerfile`, we can assure it doesn't contain any spaces or other special symbols.

Reply:
Agree.